addr_test.go

```diff
@@ -27,7 +27,7 @@ func TestAddrAddDel(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	if len(addrs) != 1 || !addr.Equal(addrs[0]) || addrs[0].Label != addr.Label {
+	if len(addrs) != 1 || !addr.Equal(addrs[0]) {
```

Comment:
seems odd to remove the address part of the test. If we really are renaming ParseAddr to ParseAddrWithLabel, we chould use ParseAddrWithLabel in the test, but I'm not convinced it needs to change